app/views/planning_applications/_planning_application_table.html.erb

```diff
@@ -36,14 +36,14 @@
           <td class="govuk-table__cell">Not required</td>
         <% elsif id == "awaiting_determination" %>
           <td class="govuk-table__cell"><%= planning_application.awaiting_determination_at.strftime("%e %b") %></td>
-        <% elsif id == "determined" %>
+        <% elsif id == "closed" %>
           <td class="govuk-table__cell"><%= planning_application.determined_at.strftime("%e %b") %></td>
         <% end %>
         <td class="govuk-table__cell">
           <% if planning_application.user %>
             <%= planning_application.user.name %>
           <% else %>
-            <strong class="govuk-tag govuk-tag--purple">Not started</strong>
+            <strong class="govuk-tag govuk-tag--purple">Not assigned</strong>
```

Comment:
Can we ask Martyn about making this not a purple tag? I think it suggests it's a state, which it isn't really.

Reply:
Martyn agreed, so this is now an empty string and the spec has been removed.